src/App.svelte

```diff
@@ -288,6 +294,30 @@
   		height: 3rem;
 		background-color: var(--header);
 		color: var(--background);
+		transition: 0.2s;
+		
+	}
+
+	.register-button:hover{
```

Comment:
I dont think you need to add this class, you can just apply the "link" class to the button below

Reply:
Th "link" class removes the original background color of the button and makes them similar to the header buttons. So to retain the background color while keeping transition i made a hover class instead